frontend/router/paths.ts

```diff
@@ -128,6 +128,8 @@ export default {
     }`;
   },
   FORGOT_PASSWORD: `${URL_PREFIX}/login/forgot`,
+  TWO_FACTOR_AUTHENTICATION: `${URL_PREFIX}/login/2fa`,
```

Comment:
Seems you're generally calling this "2fa" on the FE and @iansltx is calling it "mfa" on the backend.
Using "mfa" in general might be cleaner if we ever want to implement additional factors in the
future? Not a deal breaker.

Reply:
Eh, I see Figma calls it 2-factor all over the place, nevermind